credential-service.js

```diff
@@ -97,7 +94,7 @@ function factory($rootScope, brRefreshService, brResourceService, config) {
     // TODO: register and find displayers using priority
     // Possibly use displayer 'q' param like Accept?
     // May want to be able to override
-    types = _.reject(types, function(type) { return type === 'Credential'; });
+    types = _.reject(types, function(type) { return type === 'Credential';});
```

Comment:
Weird: A space after the opening `{` but no space before the closing `}`? Should be consistent.